docs/Installation.md

```diff
@@ -97,11 +97,23 @@ sudo install -v -m 755 ./zm-testagent.lsb /etc/init.d/zm-testagent
 sudo install -v -m 755 ./tmpfiles.conf /usr/lib/tmpfiles.d/zonemaster.conf
 ```
 
+> If this is an update of Zonemster-Backend, you should remove any
+> `/etc/init.d/zm-backend.sh` and `/etc/init.d/zm-centos.sh` (scripts from
+> previous version of Zonemaster-Backend).
+
 ### 3.2 Database engine installation and configuration (CentOS)
 
 Check the [declaration of prerequisites] to make sure your preferred combination
 of operating system version and database engine version is supported.
 
+The installation instructions below assums that this is a new installation.
+If you upgrade and want to keep the database, go to section
+[7.4](#74-upgrade-zonemaster-database) first. If you instead want to start
+from afresh, then go to section [7.3](#73-cleaning-up-the-database) and remove
+the old database first.
+
+If you keep the database, skip the initialization of the Zonemaster database,
+but if you have removed the old Zonemaster database, then do the inialization.

```

Comment:
"inialization" -> "initialization"

There are more of these.

Reply:
Fixed * 3.